paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -139,6 +138,40 @@ public class CoreOptions implements Serializable {
                             "Default file compression format, can be overridden by "
                                     + FILE_COMPRESSION_PER_LEVEL.key());
 
+    public static final ConfigOption<FileFormatType> CHANGELOG_FORMAT =
+            key("changelog.format")
+                    .enumType(FileFormatType.class)
+                    .defaultValue(FileFormatType.ORC)
+                    .withDescription(
+                            "Specify the message format of changelog files, currently orc, parquet and avro are supported.");
+
+    public static final ConfigOption<String> CHANGELOG_COMPRESSION =
+            key("changelog.compression")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("Default changelog compression format.");
+
+    public static final ConfigOption<Map<String, String>> CHANGELOG_COMPRESSION_PER_LEVEL =
```

Comment:
I don't want to introduce per level for changelog.
Can we just introduce unified format and compression?

Reply:
It depends on whether any compaction happens on the changelog and how small a file there is if you don't compate. I mean, changelog doesn't need to sort even when it does small file merges because it's in chronological order.

Reply:
Yes, so there should be no per level